azext_iot/digitaltwins/_help.py

```diff
@@ -113,6 +113,20 @@ def load_digitaltwins_help():
             az dt delete -n {instance_name} -y --no-wait
     """
 
+    helps["dt reset"] = """
+        type: command
+        short-summary: Reset an existing Digital Twins instance by deleting associated
+            dataplane assets. Currently only supports deleting models and twins.
```

Comment:
We should remove usage of "dataplane" its not necessarily a well known term publicly.